### PR TITLE
[pricing] adjust wording & delete "student" button

### PR DESCRIPTION
**Pull request by ChristinFrohne**
Messaging for Students subscriptions seemed to be too complicated. 

Button had no purpose

### CODE REVIEW

File: src/pages/pricing.tsx [hunk 1]
```diff
@@ -223,22 +223,27 @@ const PricingPage: React.SFC<{}> = () => (
...
-                            <h2>Open Source</h2>
+                            <h2>Unlimited Open Source</h2>
                             <p>
-                                Gitpod is free for up to 50 hours/month on any public repository. Just prefix any public GitHub or GitLab URL with <span className="text-with-galaxy-bg">gitpod.io/#</span> and start coding.
+                                Gitpod is free for up to 50 hours/month on any public repository. 
                             </p>
                             <p>
-                                If you're a professional open-source developer and think you need more, let us know, by applying for <strong>a free unlimited open source plan</strong>. Your GitHub / GitLab track record should clearly show your regular open-source contributions.
+                                If you're a professional open-source developer and need more, let us know, by applying for a <strong> free Unlimited Open Source Plan</strong>. Your GitHub / GitLab track record should clearly show your regular open-source contributions.
                             </p>
```
Comment (ChristinFrohne) [inline src/pages/pricing.tsx]: I'd like to link this to a section in docs explaining the requirements for it. Then the sentence about the track records could be moved there

File: src/pages/pricing.tsx [hunk 2]
```diff
@@ -223,22 +223,27 @@ const PricingPage: React.SFC<{}> = () => (
...
-                                Students get the <strong>Unlimited plan for {isEurope() ? '€8' : '$9'}</strong> per month.
+                                Get the <strong> <a href="https://gitpod.io/subscription/"> Student Unlimited Plan </a>for {isEurope() ? '€8' : '$9'}</strong> per month.
                             </p>
-                            <p>Make sure your student email is set as your primary email on GitHub. And please let us know if your educational institution is not correctly recognized.</p>
-                            <a href="/contact/" className="btn">I'm a Student</a>
+                            <p>
+                                Make sure your student email is set as your primary email on GitHub/GitLab. Please let us know if your educational institution is not correctly recognized.
+                            </p> 
+                            <p>
+                                <a href="/contact/" className="btn">Register Institution</a>
```
Comment (ChristinFrohne) [inline src/pages/pricing.tsx]: I didn't know how to do link it directly to the mail account

File: src/pages/contact.tsx [hunk 3]
```diff
@@ -220,9 +220,9 @@ export default function ContactPage() {
                                 id="eMail" 
                                 onChange={handleChange} 
                             />
-                            <label className="visually-hidden" htmlFor="message">What’s on your mind?</label>
+                            <label className="visually-hidden" htmlFor="message">What'd you like to discuss?</label>
                             <textarea name="message" className="form__textarea"
-                                placeholder='What’s on your mind?'
+                                placeholder='What’d you like to discuss?'
```
Comment (jankeromnes) [inline src/pages/contact.tsx]: Alternatively, we could also remove the `placeholder` entirely.

I think a blank textarea in a Contact form is understandable enough ("ok, that's where I write my contact message"). Any specific placeholder risks altering the answers (e.g. here I imagine some people will just write "Opportunities with Gitpod", because that answers the question).

EDIT: Or at least, I think it should not be a question, because that prompts users to just write a brief answer.

File: src/pages/pricing.tsx [hunk 4]
```diff
@@ -223,22 +223,27 @@ const PricingPage: React.SFC<{}> = () => (
 
                     <section className="links">
                         <div>
-                            <h2>Open Source</h2>
+                            <h2>Unlimited Open Source</h2>
                             <p>
-                                Gitpod is free for up to 50 hours/month on any public repository. Just prefix any public GitHub or GitLab URL with <span className="text-with-galaxy-bg">gitpod.io/#</span> and start coding.
+                                Gitpod is free for up to 50 hours/month on any public repository.
```
Comment (jankeromnes) [inline src/pages/pricing.tsx]: I think we don't need to repeat the 50 hours in the first sentence. Maybe just something like:

> Gitpod.io is free to use on any public repository.
> 
> If you're a professional open source developer, and you need more than 50 hours every month, you can also apply to our Unlimited Open Source plan.
> [Apply Now]

File: src/pages/pricing.tsx (hunk 1 above)
Comment (jankeromnes) [reply src/pages/pricing.tsx]: A new section in the docs could be helpful, e.g. under https://www.gitpod.io/docs/subscriptions/

Also, the "track records" sentence doesn't seem very specific to me. Better mention the full criteria explicitly.

File: src/pages/pricing.tsx [hunk 5]
```diff
@@ -223,22 +223,27 @@ const PricingPage: React.SFC<{}> = () => (
...
-                                If you're a professional open-source developer and think you need more, let us know, by applying for <strong>a free unlimited open source plan</strong>. Your GitHub / GitLab track record should clearly show your regular open-source contributions.
+                                If you're a professional open-source developer and need more hours, let us know, by applying for a <strong> free Unlimited Open Source Plan</strong>. Your GitHub / GitLab track record should clearly show your regular open-source contributions.
                             </p>
                             <a href="/contact/" className="btn">Apply Now</a>
                         </div>
                         <div>
                             <h2>For Students</h2>
                             <p>
-                                Students get the <strong>Unlimited plan for {isEurope() ? '€8' : '$9'}</strong> per month.
+                                Get the <strong> <a href="https://gitpod.io/subscription/"> Student Unlimited Plan </a>for {isEurope() ? '€8' : '$9'}</strong> per month.
```
Comment (jankeromnes) [inline src/pages/pricing.tsx]: I think the word "plan" should not be part of the highlighted plan name (also, `target="_blank"` allows opening in a separate tab):

```suggestion
                                Get the <strong><a href="https://gitpod.io/subscription/" target="_blank">Student Unlimited</a> plan for {isEurope() ? '€8' : '$9'}</strong> per month.
```

File: src/pages/pricing.tsx (hunk 2 above)
Comment (jankeromnes) [reply src/pages/pricing.tsx]: Like this:

```suggestion
                                <a href="mailto:contact@gitpod.io?subject=Student Unlimited" className="btn">Register Institution</a>
```

Comment (jankeromnes) [reply src/pages/pricing.tsx]: But I think that `Register Institution` gives the wrong expectation: When reading this, I expect to be taken to a form, where I can provide details about my University, and officially register it to Gitpod. Also, since this seems complicated and potentially dangerous, I won't take the risk of clicking.

This button is good for students who don't see the Student Unlimited plan, but are not exactly sure why: Maybe they don't have a student email address? Or they can't use it on GitHub? Or they're not sure if their coding bootcamp even qualifies as an "educational institution"? I think there are other use cases than just registering a new university domain (even if that's most of what we do).

Maybe change the button to `Enable Student Status`? Or `Get Verified as Student`? Or `Apply for Student Unlimited`?

File: src/pages/pricing.tsx (hunk 1 above)
Comment (ChristinFrohne) [reply src/pages/pricing.tsx]: I created an issue for it https://github.com/gitpod-io/website/issues/453

File: src/pages/pricing.tsx (hunk 2 above)
Comment (ChristinFrohne) [reply src/pages/pricing.tsx]: I agree. I like `Get Verified as Student`

File: src/pages/contact.tsx (hunk 3 above)
Comment (ChristinFrohne) [reply src/pages/contact.tsx]: I think having it empty is weird, as the other fields have a description. 
What about simply "Your Message" / "Please enter your message"? @jankeromnes

File: src/pages/contact.tsx [hunk 6]
```diff
@@ -220,9 +220,9 @@ export default function ContactPage() {
                                 id="eMail" 
                                 onChange={handleChange} 
                             />
-                            <label className="visually-hidden" htmlFor="message">What'd you like to discuss?</label>
+                            <label className="visually-hidden" htmlFor="message">Please Enter Your Message</label>
```
Comment (jankeromnes) [inline src/pages/contact.tsx]: Super-small suggestion, I feel like "Enter" has a very small chance of meaning "Submit", so I'd personally prefer the word "type" here. Also maybe no caps (feels more natural to read, and it's not really a title or button text):

```suggestion
                            <label className="visually-hidden" htmlFor="message">Please type your message</label>
```

File: src/pages/contact.tsx [hunk 7]
```diff
@@ -220,9 +220,9 @@ export default function ContactPage() {
...
                             <textarea name="message" className="form__textarea"
-                                placeholder='What’d you like to discuss?'
+                                placeholder='Please Enter Your Message'
```
Comment (jankeromnes) [inline src/pages/contact.tsx]: Same as above:

```suggestion
                                placeholder='Please type your message'
```

File: src/pages/pricing.tsx (hunk 1 above)
Comment (jankeromnes) [reply src/pages/pricing.tsx]: Awesome, thanks! I think this is resolved then.

File: src/pages/pricing.tsx [hunk 8]
```diff
@@ -228,7 +228,7 @@ const PricingPage: React.SFC<{}> = () => (
                                 Gitpod is free for up to 50 hours/month on any public repository. 
                             </p>
                             <p>
-                                If you're a professional open-source developer and need more hours, let us know, by applying for a <strong> free Unlimited Open Source Plan</strong>. Your GitHub / GitLab track record should clearly show your regular open-source contributions.
+                                If you're a professional open-source developer and need more hours, you can apply to our <strong> free Unlimited Open Source Plan</strong>.
```
Comment (jankeromnes) [inline src/pages/pricing.tsx]: Please don't add extra spaces within tags like `<strong>` and `</strong>`, because this can lead to double-spaces in the text (there are already spaces outside of these tags).

Also, I think it's helpful to clearly call out the full **Unlimited Open Source** plan name, without any extra words in bold or caps, so that you very visually see & remember exactly what you want (i.e. not just "the open source plan" as many users currently request, or "hey I want to use Gitpod for open source thanks", etc).

In summary:

```suggestion
                                If you're a professional open-source developer and need more hours, you can apply to our free <strong>Unlimited Open Source</strong> plan.
```

File: src/pages/pricing.tsx (hunk 4 above)
Comment (jankeromnes) [reply src/pages/pricing.tsx]: @ChristinFrohne this comment is still open and unresolved. Please decide to either address it (i.e. apply suggestion) or discard it (i.e. reply why you don't like the suggestion), and then you can "resolve" it on GitHub. 🙂


FYI it's totally okay to object to review comments, e.g. "no I like my version better thanks". 👍

File: src/pages/contact.tsx (hunk 3 above)
Comment (jankeromnes) [reply src/pages/contact.tsx]: Sure, your suggestions sound more neutral and thus won't risk distracting visitors. Thanks!

File: src/pages/pricing.tsx (hunk 4 above)
Comment (ChristinFrohne) [reply src/pages/pricing.tsx]: >I think we don't need to repeat the 50 hours in the first sentence.
this is the first time we're mentioning the 50 hours, isn't it? 
In that case I think it's good to tell honestly and directly the open-source deal. I aligned the rest of the sentence to your suggestion
